src/layouts/components/Editor/extensions/FormSG/FormSGView.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Text } from "@chakra-ui/react"
+import { NodeViewProps, NodeViewWrapper } from "@tiptap/react"
+
+export const FormSGView = ({ node }: NodeViewProps) => {
+  const formSrc = node.content.child(1).attrs.src
+
+  return (
+    <Box
+      as={NodeViewWrapper}
+      bg="#FAF594"
```

Comment:
We will be replacing the entire render in the future together with the Iframe one, because the current design was taken from Tiptap and does not match our existing design system. That will be done in a separate PR once the designs are ready.